src/sites_toolbox.py

```diff
@@ -297,3 +300,30 @@ def sites_toolbox_page():
         st.write(user, "is now in the", user_site_group, "site group.")
         if st.button("Close"):
             st.empty()
+
+    # create a new site
+    st.markdown(
+        f'<h1 style="color:#63BCAF;font-size:32px;">{"Create New Site"}</h1>',
+        unsafe_allow_html=True,
+    )
+    if st.button("Input new site data"):
+        with connection.get_session() as session:
+            client_site_id = st.text_input("Enter client_site_id")
+            client_site_name = st.text_input("Enter client_site_name")
+            latitude = st.text_input("Enter latitude")
+            longitude = st.text_input("Enter longitude")
+            capacity_kw = st.text_input("Enter capacity_kw")
```

Comment:
Could you an some optional variables, like tilt and orientation? Perhaps there should be a red start with the ones that are needed